pkg/resolver/searchSubscription.go

```diff
@@ -0,0 +1,55 @@
+// Copyright Contributors to the Open Cluster Management project
+package resolver
+
+import (
+	"context"
+	"time"
+
+	klog "k8s.io/klog/v2"
+
+	"github.com/stolostron/search-v2-api/graph/model"
+	"github.com/stolostron/search-v2-api/pkg/config"
+)
+
+func SearchSubscription(ctx context.Context, input []*model.SearchInput) (<-chan []*SearchResult, error) {
+	ch := make(chan []*SearchResult)
+
+	// You can (and probably should) handle your channels in a central place outside of `schema.resolvers.go`.
+	// For this example we'll simply use a Goroutine with a simple loop.
+	go func() {
+		timeout := time.After(time.Duration(config.Cfg.SubscriptionPollTimeout) * time.Minute)
+		// Handle deregistration of the channel here. Note the `defer`
+		defer close(ch)
+
+		for {
+			klog.Infof("Search subscription new poll interval")
```

Comment:
1. Change to `klog.Info()`.   FYI, Infof() is used when including variables in the log.
2. Increase verbosity with `V(3)` or higher.